dev-dependency-linter.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const glob = require('glob');
+
+let rootPackageJson;
+
+try {
+  rootPackageJson = JSON.parse(fs.readFileSync('./package.json'));
+} catch (e) {
+  throw new Error(`Failed to parse package.json, with error: ${e}`);
+}
+
+let workspaces;
+
+if (rootPackageJson.workspaces) {
+  workspaces = rootPackageJson.workspaces;
+} else {
+  throw new Error('It seems like the root package.json doesn\'t define a workspace.');
+}
+
+
+workspaces.forEach(workspace => {
+  const packages = glob.sync(workspace);
```

Comment:
is glob the really correct way to handle any possible string inside this key? did you check what yarn is doing?